metis-core/metis-core-service/src/main/java/eu/europeana/metis/core/service/DepublishRecordIdService.java

```diff
@@ -178,4 +189,24 @@ public int addRecordIdsToBeDepublished(MetisUser metisUser, String datasetId,
     result.setResultsAndLastPage(records, depublishRecordIdDao.getPageSize(), page);
     return result;
   }
+
+  public WorkflowExecution createDepublishWorkflowExecution(MetisUser metisUser, String datasetId,
```

Comment:
Somewhat misleading method name? Perhaps something like `scheduleDepublishWorkflowExecution` to make clear that it is not just creating but also executing?

Reply:
Renamed